AIPscan/Reporter/templates/reports.html

```diff
@@ -48,33 +47,128 @@
             <td>AIP contents</td>
             <td></td>
             <td></td>
+            <td></td>
             <td><a href="#"><button type="button" id="report2a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
         </tr>
 
         <tr>
             <td>Format version count</td>
             <td></td>
             <td></td>
+            <td></td>
             <td><a href="#"><button type="button" id="report3a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
         </tr>
 
         <tr>
             <td>Preservation copy count</td>
             <td></td>
             <td></td>
+            <td></td>
             <td><a href="#"><button type="button" id="report4a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
         </tr>
 
         <tr>
             <td>Largest files</td>
             <td></td>
             <td></td>
+            <td></td>
             <td><a href="#"><button type="button" id="report9a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
         </tr>
 
+        <tr>
+            <td>
+                File format
+                <br>
+                <span class="text-muted">AIPs containing original files in given file format.</span>
```

Comment:
I completely understand what you're getting at but I wonder if end users would be familiar with the term "format naming group", or if it might have the potential to cause confusion.

I've tried to be consistent with the FPR data model (or at least the `Format` and `FormatVersion` subset) as much as possible in this work when considering what a "format" vs. "format version" is, with the aim of at least being consistent within the Archivematica ecosystem, but I agree there are issues here, as the PDF example makes very clear. From my understanding, Archivematica/the FPR always makes identifications at the `FormatVersion` level (and that's where we find PUIDs as well, which is why I thought it would be safe to use the PUID as a proxy for file format version), and every `FormatVersion` belongs to a `Format` (as a many-to-one relationship, but in practice sometimes it's one-to-one in our data) so I've tried to stick to those terms as much as possible, even when they're not quite how I'd use them outside of this context.

That said, this is a tricky one and I don't feel very confident either way so if you have a strong leaning here I'd be happy to make the change and maybe we can try to get some user validation/feedback down the road?

Reply:
Definitely tricky but I can understand your approach. 

I'm reminding myself of trivia when someone thinks something might be the right answer, but doesn't have an answer :smile: 

I suspect this might get clearer/easier with a PRONOM-like (ish) integration in future so that's in the back of my mind. And some validation/feedback will help - if it works it works. 

My leaning is to try and do something hearing what you say about format naming group, I think maybe that won't work then. But at the same time, I'd love to not assert this aggregation as a format version. 

With your not to alignment with the FPR what about adding the PUID IDs in brackets alongside the title? 

![image](https://user-images.githubusercontent.com/1880412/100390461-6fdec080-2ffe-11eb-8fd7-77b789f35896.png)

Does that work? Maybe we can each chat offline too if it doesn't quite hit the mark.

Reply:
We discussed a bit offline and, although I like the idea, adding the PUIDs in brackets might introduce some new UI issues and might be tricky with the data that we currently have in AIPscan, so we will leave this as-is for now and I will file a new issue describing the problem and some of our thinking to date on it.